drivers/integration/docker/docker.go

```diff
@@ -391,41 +506,41 @@ func (d *driver) NetworkName(
 }
 
 func (d *driver) volumeRootPath() string {
-	return d.config.GetString("linux.volume.rootPath")
+	return d.config.GetString(ConfigDockerLinuxVolumeRootPath)
 }
 
 func (d *driver) volumeType() string {
-	return d.config.GetString("docker.volumeType")
+	return d.config.GetString(ConfigDockerVolumeType)
 }
 
 func (d *driver) iops() string {
-	return d.config.GetString("docker.iops")
+	return d.config.GetString(ConfigDockerIOPS)
 }
 
 func (d *driver) size() string {
-	return d.config.GetString("docker.size")
+	return d.config.GetString(ConfigDockerSize)
 }
 
 func (d *driver) availabilityZone() string {
-	return d.config.GetString("docker.availabilityZone")
+	return d.config.GetString(ConfigDockerAvailabilityZone)
 }
 
 func (d *driver) fsType() string {
-	return d.config.GetString("docker.fsType")
+	return d.config.GetString(ConfigDockerFsType)
 }
 
 func (d *driver) mountDirPath() string {
-	return d.config.GetString("docker.mountDirPath")
+	return d.config.GetString(ConfigDockerMountDirPath)
 }
 
 func configRegistration() *gofig.Registration {
 	r := gofig.NewRegistration("Docker")
-	r.Key(gofig.String, "", "ext4", "", "docker.fsType")
-	r.Key(gofig.String, "", "", "", "docker.volumeType")
-	r.Key(gofig.String, "", "", "", "docker.iops")
-	r.Key(gofig.String, "", "", "", "docker.size")
-	r.Key(gofig.String, "", "", "", "docker.availabilityZone")
-	r.Key(gofig.String, "", "/var/lib/libstorage/volumes", "", "docker.mountDirPath")
-	r.Key(gofig.String, "", "/data", "", "linux.volume.rootpath")
+	r.Key(gofig.String, "", "ext4", "", ConfigDockerFsType)
+	r.Key(gofig.String, "", "", "", ConfigDockerVolumeType)
+	r.Key(gofig.String, "", "", "", ConfigDockerIOPS)
+	r.Key(gofig.String, "", "16", "", ConfigDockerSize)
+	r.Key(gofig.String, "", "", "", ConfigDockerAvailabilityZone)
+	r.Key(gofig.String, "", "/var/lib/libstorage/volumes", "", ConfigDockerMountDirPath)
+	r.Key(gofig.String, "", "/data", "", ConfigDockerLinuxVolumeRootPath)
```

Comment:
Hi @clintonskitson,

So there's no need to register the old keys via Gofig unless you want them to be available as environment variables and command line flags. Simply checking them via the `Get___` functions is sufficient. Those properties **will** be parsed if in a config file, whether registered or not. The only thing registering them does is make them available as environment variables and command line flags.